### PR TITLE
remove all backends but MKL

### PR DESCRIPTION
**Pull request by massich**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,65 +3,29 @@ sudo: true
...
-    - os: osx
-      env:
-        - BACKEND=Intel10_64lp
-    - os: osx
-      env:
-        - BACKEND=Intel10_64lp
-        - BUILD_STATIC=1
-    - os: osx
-      env:
-        - BACKEND=OpenBLAS
```
Comment (agramfort) [inline .travis.yml]: please put back the mac mkl builds

File: .appveyor.yml [hunk 2]
```diff
@@ -35,8 +35,8 @@ environment:
       BUILD_STATIC : OFF
 
 
-# init:
-#   - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+init:
+  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```
Comment (agramfort) [inline .appveyor.yml]: why did you add these lines? I removed them in my merged PR

File: .travis.yml (hunk 1 above)
Comment (agramfort) [reply .travis.yml]: piing.

please put back the mac builds here so we can move on.

Comment (agramfort) [reply .travis.yml]: but not the OpenBLAS one.

File: .appveyor.yml (hunk 2 above)
Comment (massich) [reply .appveyor.yml]: These lines are those that allow to connect to appveyor remotelly. What I do usually is that they are there commented, and I do a commit uncommenting them. When I'm happy, I remove this commit from the history.

**Comment by agramfort** [comment]
shall I remove them now then?

File: .travis.yml (hunk 1 above)
Comment (massich) [reply .travis.yml]: my idea was that once everything was green remove the `aed69c6` commit.

**Comment by agramfort** [comment]
everything was green :)

**Comment by massich** [comment]
@agramfort sure.
